show/muxcable.py

```diff
@@ -336,3 +339,43 @@ def config(port, json_output):
             click.echo(tabulate(print_data, headers=headers))
 
         sys.exit(CONFIG_SUCCESSFUL)
+
+
+@muxcable.command()
+@click.argument('port', required=True, default=None, type=click.INT)
+@click.argument('target', required=True, default=None, type=click.INT)
+def berinfo(port, target):
+
```

Comment:
Please add a docstring to be used as a brief help message

Reply:
fixed